python/ray/tune/tune.py

```diff
@@ -24,6 +25,55 @@
 }
 
 
+class Experiment():
+    """Tracks experiment specifications.
+
+    Parameters:
+        name (str): Name of experiment.
+        run (str): The algorithm or model to train. This may refer to the
+            name of a built-on algorithm (e.g. RLLib’s DQN or PPO), or a
+            user-defined trainable function or class
+            registered in the tune registry.
+        stop (dict): The stopping criteria. The keys may be any field in
+            TrainingResult, whichever is reached first.
+        config (dict): Algorithm-specific configuration
+            (e.g. env, hyperparams).
+        resources (dict): Machine resources to allocate per trial,
+            e.g. ``{"cpu": 64, "gpu": 8}``. Note that GPUs will not be
+            assigned unless you specify them here. Defaults to 1 CPU and 0
+            GPUs.
+        repeat (int): Number of times to repeat each trial. Defaults to 1.
+        local_dir (str): Local dir to save training results to.
```

Comment:
If possible, would it be better to call this `logdir` rather than `local_dir` for consistency with the `Trainable` class?

Reply:
Let me make a separate issue for this, as it is a bit of a more involved refactor